calver/compare.go

```diff
@@ -119,3 +119,205 @@ func (c *Version) CompareOrPanic(other *Version) int {
 	}
 	return compare
 }
+
+// Equal returns true if the versions are equal, false otherwise.
+// If Compare() returns an error, Equal will propagate it.
+// Example:
+//
```

Comment:
```suggestion
// If Compare() returns an error, Equal will propagate it.
//
// Example:
//
```

pls add a new line before the example